src/site/markdown/server/manage-webadmin.md

```diff
@@ -2204,10 +2204,10 @@ concurrent changes done during the reIndexing might be ignored.
 ### ReIndexing a mailbox mails
 
 ```
-curl -XPOST http://ip:port/mailboxIndex/users/bob@domain.com/mailboxes/{mailboxId}?task=reIndex
+curl -XPOST http://ip:port/mailboxIndex/mailboxes/{mailboxId}?task=reIndex
```

Comment:
I'm still convinced that mailboxIndex should not be a visible resource outside of James. Why don't we implement this action on `/mailboxes/` ?

Reply:
* all mailboxes is easy, you omit the mailboxid and that's all
* per user is more tricky, you can pass it as a query string (filters are usually query params in REST)

Reply:
Maybe such API changes could be handled in a separate PR?